src/features/content-explorer/content-explorer/__tests__/ContentExplorerInfoNotice.test.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+import ContentExplorerInfoNotice from '../ContentExplorerInfoNotice';
+
+describe('features/content-explorer/content-explorer/ContentExplorerInfoNotice', () => {
+    const renderComponent = infoNoticeText => shallow(<ContentExplorerInfoNotice infoNoticeText={infoNoticeText} />);
```

Comment:
new tests should use react testing library

Reply:
Updated